### PR TITLE
Implement oms3_import

### PR DESCRIPTION
**Pull request by lochel**
### Purpose

Implement oms3_import

### Type of Change

- New feature (non-breaking change which adds functionality)

### Open Questions and Pre-Merge TODOs

-  [x] Import ssd:SimulationInformation
-  [x] Import ssd:Connectors
-  [x] Import ssd:Connections

### CODE REVIEW

**Comment by lochel** [comment]
@adeas31 @robbr48 Do you think it is a good way of implementing the import feature? I am a bit uncertain if the code can be easily extended and maintained.

File: src/OMSimulatorLib/System.cpp [hunk 1]
```diff
@@ -259,6 +259,64 @@ oms_status_enu_t oms3::System::exportToSSD(pugi::xml_node& node) const
...
+            return oms_status_error;
+
+          if (oms_status_ok != system->importFromSSD(*itElements))
+            return oms_status_error;
+        }
+        else
+          return logError("wrong xml schema detected: " + name);
+      }
+    }
+    else
```
Comment (robbr48) [inline src/OMSimulatorLib/System.cpp]: I think we should also load system annotations here.

Comment (lochel) [reply src/OMSimulatorLib/System.cpp]: Do we already export any system annotations?

Comment (robbr48) [reply src/OMSimulatorLib/System.cpp]: Not yet, but we will soon need them for bus connections.

**Comment by OpenModelica-Hudson** [comment]
The following pull requests are now being [tested](https://test.openmodelica.org/hudson/job/OMSimulator-PR/638//console). It may take a while, depending on the workload.
  - OpenModelica/OMSimulator#376
  - OpenModelica/OMSimulator-testsuite#84

The changes will be pushed if all tests run successfully.

**Comment by OpenModelica-Hudson** [comment]
All tests were successful. :tada:

**Pull request closed without merging**